types/multiset/fundamental/README.md

```diff
@@ -0,0 +1,15 @@
+# `std::multiset<std::int32_t>`
+
+## Fields
+
+ * `[Split]Index{32,64}`
+
+with the corresponding column type for the offset column of the collection parent field.
+All child fields use the default column encoding `Int32`.
+
+## Entries
+
+1. Single-element sets, with ascending values
+2. Empty sets
+3. Increasing number of elements in the set:
+   one element in the first field, two elements in the second field, etc.
```

Comment:
I see you removed the duplicate elements in the latest push. Sorry, this was not precise from my side: I think for `[unordered_]multiset` we probably want them...

Reply:
Hmmm why? In the end that would also just be testing the semantics of the C++ container which is precisely the reason we decided to remove it for the other `set` variations.

Reply:
Hm, for `std::set` duplicate values are "filtered" on the C++ side before they even reach RNTuple. As a result, they don't end up on disk and therefore cannot be read back. For `std::multiset`, the duplicate values are stored on disk and should be read back, which I think would be useful to exercise.

Reply:
I'm fine adding them back, but I do think the the definition of what we actually want to test is too blurry at this point. Still, the duplication is managed/"kept" by the in-memory representation of the type, and the on-disk representation is identical to that of a `std::vector` so more than anything we would be testing how a reader/writer extracts/constructs data from/into the in-memory object (i.e. in ROOT's case through collection proxies).

I'm not saying this is something we _don't_ want to do, but based on (my current interpretation of) the objectives of this validation suite, the decision to add or not add certain tests to me seem to go beyond these objectives.

Reply:
Just to write down what I said during the IO meeting: I agree that the semantic data on disk is identical to `std::vector`, so it's indeed not a matter of the binary format. I'm resolving this conversation.